packages/x-components/.eslintrc.js

```diff
@@ -8,5 +8,13 @@ module.exports = {
   rules: {
     'no-dupe-class-members': 'off',
     '@typescript-eslint/no-unused-vars-experimental': 'off'
-  }
+  },
+  overrides: [
```

Comment:
Right now if you want to put a `d='23123 213123'` in multiple lines, you have to do it manually, so this is a little bit frustrating. Thats why he added this rule to avoid linting the icons manually. I dont know if there if something to break the SVG code in multiple lines